### PR TITLE
doc: add doc about run on aws eks

### PR DESCRIPTION
**Pull request by tokers**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

___
### Backport patches
- Why need to backport?

- Source branch

- Related commits and pull requests

- Target branch

### CODE REVIEW

**Comment by tokers** [comment]
@membphis @moonming @gxthrj Please take a look when you have time.

**Comment by moonming** [comment]
great， we can post to apisix.apache.org too

**Review by membphis — APPROVED** [review]

File: docs/deployments/aws.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+```shell
+cd /path/to/apisix-ingress-controller
+# install base resources, e.g. ServiceAccount.
+helm install ingress-apisix-base -n ingress-apisix ./charts/base
+# install apisix-ingress-controller
+helm install ingress-apisix ./charts/ingress-apisix \
+  --set ingressController.image.tag=dev \
+  --set ingressController.config.apisix.baseURL=http://apisix-admin:9180/apisix/admin \
+  --namespace ingress-apisix
+```
```
Comment (gxthrj) [inline docs/deployments/aws.md:71]: Need to set `admin_key`` when use helm install.

Comment (tokers) [reply docs/deployments/aws.md:71]: OK, actually the default admin_key is in use, but it's really a good idea to set it explicitly.

File: docs/deployments/aws.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+```shell
+cd /path/to/apisix-ingress-controller
+# install base resources, e.g. ServiceAccount.
+helm install ingress-apisix-base -n ingress-apisix ./charts/base
+# install apisix-ingress-controller
+helm install ingress-apisix ./charts/ingress-apisix \
+  --set ingressController.image.tag=dev \
+  --set ingressController.config.apisix.baseURL=http://apisix-admin:9180/apisix/admin \
+  --set ingressController.config.apisix.adminKey=edd1c9f034335f136f87ad84b625c8f1
```
Comment (gxthrj) [inline docs/deployments/aws.md:70]: need  a \ in the end

Comment (tokers) [reply docs/deployments/aws.md:70]: OK

Comment (tokers) [reply docs/deployments/aws.md]: Fixed.

**Review by gxthrj — APPROVED** [review]

**Pull request merged**